examples/tts/conf/hifigan/hifigan.yaml

```diff
@@ -36,23 +36,31 @@ model:
     lr: 0.0002
     betas: [0.8, 0.99]
 
+  sched:
+    name: CosineAnnealing
+    min_lr: 1e-5
+    warmup_ratio: 0.02
+    max_steps: ???
+
   denoise_strength: 0.0025
-  max_steps: 2500000
 
 trainer:
   gpus: -1 # number of gpus
-  max_steps: ${model.max_steps}
+  max_epochs: ???
```

Comment:
Why switch this?

Reply:
Because it is not valid code

Reply:
What do you mean by it is not valid code? What was the error?

Reply:
I see, ok that is an error on my part. I will fix it

Reply:
@blisc it works!